api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxMlCohortQueries.java

```diff
@@ -1468,15 +1471,15 @@ public CohortDefinition getPatientsIITLessThan90DaysComposition() {
     cd.addSearch(
         "dead",
         EptsReportUtils.map(
-            getDeadPatientsComposition(),
+            getPatientsWhoMissedNextAppointmentAndDiedDuringReportingPeriod(),
             "startDate=${startDate},endDate=${endDate},location=${location}"));
     cd.addSearch(
         "transferredOut",
         EptsReportUtils.map(
             getPatientsWhoMissedNextAppointmentAndTransferredOut(),
             "startDate=${startDate},endDate=${endDate},location=${location}"));
 
-    cd.setCompositionString("(missedAppointment AND C1) AND NOT dead AND NOT transferredOut");
+    cd.setCompositionString("(missedAppointment AND C1) AND NOT dead ANd NOT transferredOut");
```

Comment:
maybe we need to maintain the expressions key works keep it to upper case only, you have a mix here I can see **And**

Reply:
Done